generator/types.go

```diff
@@ -709,10 +710,24 @@ func NewField(n, t string, tag reflect.StructTag) *Field {
 		primaryKey:      pkName,
 		columnName:      columnName(n, tag),
 		isPrimaryKey:    isPrimaryKey,
+		isUnique:        uniqueProperty(tag),
 		isAutoincrement: autoincr,
 	}
 }
 
+func uniqueProperty(tag reflect.StructTag) bool {
+	val, ok := tag.Lookup("unique")
```

Comment:
you can go with `tag.Get("unique") == "true"`, all other options would be ignored until we decide if it's worth implementing the custom name